en/docs/index.md

```diff
@@ -502,7 +502,7 @@ template: templates/single-column.html
                                     </div>
                                 </div>
                                 <hr/>
-                                <p class="removeTopMargin">An API Gateway for micro services, which is cloud-native, decentralized and developer centric</p>
```

Comment:
This line refers to WSO2 Immutable Gateway and follows a certain pattern of sentence structure from other cards. Therefore better to keep the previous sentence.